app/src/main/java/ch/epfl/polybazaar/chat/ChatActivity.java

```diff
@@ -52,15 +58,27 @@
     private List<ChatMessage> conversation = new ArrayList<>();
 
     private FCMServiceAPI fcmServiceAPI;
+    private BroadcastReceiver broadcastReceiver;
 
     public static final String bundleListingId = "listingID";
     public static final String bundleReceiverEmail = "receiverEmail";
+    public static final String broadcastUpdateMessage = "broadcastUpdateMessage";
+    public static final String currentChatID = "currentChatID";
+    public static final String noCurrentID = "noCurrentID";
```

Comment:
These should probably be in `UPPER_CASE` since they are constants.

Reply:
Done